### DISCUSSION TITLE
Cannot build MahApps.Metro.Demo

### DISCUSSION THREAD

**Discussion by Benavolente** [MahApps/MahApps.Metro#4260] [Q&A] [answered]
Hi! I wanted to have a look at the source code of the demo, so I pulled the develop branch and tried to build it. However, I am getting a bunch of errors of this type:

CSC : error CS8032: An instance of analyzer WpfAnalyzers.AttributeAnalyzer cannot be created from \.nuget\packages\wpfanalyzers\4.0.1\analyzers\dotnet\cs\WpfAnalyzers.dll : Could not load file or assembly 'Microsoft.CodeAnalysis, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies.

I have been trying for a while to get it working but no success so far. Does anyone know I can fix this?

Thanks!

**Comment by punker76** [comment, marked as answer]
@Benavolente Which Visual Studio are you using?

Please try also to build the project by using the command line ps:

```ps
.\build.ps1
```

at the root folder.
